stle.css

```diff
@@ -19,37 +19,38 @@ header {
 
 .nav-bar .name {
   margin-left: 2.2rem;
-  font-family: "Roboto", sans-serif;
-  font-style: normal;
-  font-weight: bold;
-  font-size: 18px;
-  line-height: 24px;
 }
 
 .nav-bar .menu {
   margin-right: 2.2rem;
 }
 
 .mobile-menu {
+  margin-top: 650px;
+  padding: 0;
   display: none;
-  top: 0;
-  position: fixed;
+  position: absolute;
   width: 100%;
-  padding: 0;
   z-index: 232;
   background-color: #3c3a39;
-  height: 100%;
+  height: 700px;
   transition: 0.5s ease-in;
 }
 
 .mobile-menu a {
   text-decoration: none;
+  color: white;
+}
+
+.menu-text a {
+  text-decoration: none;
+  color: white;
 }
 
 .close-icon {
   float: right;
   margin-right: 40px;
-  margin-top: 20px;
+  margin-top: 7px;
   width: 4%;
 }
```

Comment:
- [ ] Good job working on the styles but still there is a little issue. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible. Your design is a little different from the template. Please fix the close X button alignment for both desktop and mobile screens, Increase size of the button and make sure it matches the design.

Your design

![Screenshot (191)](https://user-images.githubusercontent.com/83803414/156425964-5d1dbc7b-0870-449b-ad79-b2ef0945f781.png)


Figma design

![Screenshot (190)](https://user-images.githubusercontent.com/83803414/156425920-0933617f-6262-4f85-835a-7a40fa8fe97f.png)